vllm/entrypoints/openai/rpc/__init__.py

```diff
@@ -9,6 +9,7 @@
 
 VLLM_RPC_SUCCESS_STR = "SUCCESS"
 VLLM_RPC_HEALTHY_STR = "HEALTHY"
+VLLM_RPC_PATH = "ipc:///tmp/vllm_rpc_path"
```

Comment:
I think `zmq` should have some utilities similar to binding to a random port. If not, we can randomly generate a uuid for the path. As @njhill mentioned, we might have more than one vLLM instance running in the same node.

Reply:
sounds good